src/bin/club-story

```diff
@@ -182,10 +190,22 @@ const main = async () => {
     return;
 };
 
+const downloadFiles = (story) => {
+    story.files.map(file => {
+        const filePath = path.join(program.downloadDir ? program.downloadDir : '.', file.name);
```

Comment:
Instead of using a ternary operator here, we could simply specify the `'.'` as the default value for the `--download-dir` option (the third argument in the option definition call) on line 28.